### ISSUE TITLE
Failed to run shell file "convert.sh"

### ISSUE THREAD

**Issue by zhaoming0** [mil-tokyo/webdnn#707]
When running shell file [convert.sh](https://github.com/mil-tokyo/webdnn/blob/master/example/benchmark/convert.sh) found 2 issues:

1. **line 6**
```
python generate_keras_models.py
``` 
update to 
```
python3 generate_keras_models.py
```
Should we change all of **python --> python3 ?** Since the file `generate_keras_models.py` is edited via python3

2. **line 12**
```
git checkout --hard e0e6bd0ba9d74debcf4654697d5866169d3eb3e7
```
update to 
```
cd kerasjs && git reset --hard e0e6bd0ba9d74debcf4654697d5866169d3eb3e7 && cd -
```
`git checkout --hard`  is invalid

**Comment by Kiikurage** [comment]
Sorry, I'll fix them soon.

**Comment by Kiikurage** [comment]
Thank you for sendin PR. This problem is fixed in #710.
